CONTRIBUTING.md

```diff
@@ -23,4 +23,4 @@ If you are contributing a new hunter:
 1. When you open an issue to present the hunter, please specify which `Vulnerability` classes you plan to add.
 1. A maintainer will assign each `Vulnerability` a VID for you to include in your hunter code.
 1. Please add a KB article to `/docs/kb/` explaining the vulnerability and suggesting remediation steps. Look at other articles for examples.
-1. Please adhere to the following types convention: Use `Hunter` class to report vulnerabilities, `ActiveHunter` if your Hunter might change the state of the cluster, and `Discovery` for scanning the cluster (all are descendants of `HunterBase`). Also, use the `Vulnerability` class to report findings, and `Service` to report a discovery to be used by a hunter (both are descendants of `Event`, refrain from using `Event` directly).
\ No newline at end of file
+1. Please adhere to the following types convention: Use `Hunter` class to report vulnerabilities, `ActiveHunter` if your Hunter might change the state of the cluster, and `Discovery` for scanning the cluster (all are descendants of `HunterBase`). Also, use the `Vulnerability` class to report findings, and `Service` to report a discovery to be used by a hunter (both are descendants of `Event`, refrain from using `Event` directly).
```

Comment:
What is the change here?
It looks identical

Reply:
I don't think that I have changed anything there. But it still shows it has some diff.